src/aims_ui/models/get_addresses.py

```diff
@@ -0,0 +1,23 @@
+from .address import Address
+
+
+def get_addresses(json_response, called_from):
+  """Create Address classes for each address, might need to be different for each endpoint"""
+  addresses = []
+
+
+  if called_from == 'uprn':
+    response = (json_response.get('response'))
+    address = response.get('address')
+    addresses=[Address(address)]
+  
+  elif (called_from == 'postcode') or (called_from == 'singlesearch'):
+    response = (json_response.get('response'))
+    address_json = response.get('addresses')
+
+    for address in address_json:
+      addresses.append(Address(address))
```

Comment:
It doesn't seem that any verification is done on addresses that enter this way. This might be okay, or it could result in some errors slipping through.

Reply:
I'll  get a ticket done to wrap all specific errors from the api, but that should return an error if the address is wrong